task/index.html

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>js 1st task</title>
-</head>
-<body>
-    <script src="main.js"></script>
```

Comment:
Directly add it in html (order is important).
<script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>

    <script src="./assets/js/utils.js"></script>
    <script src="./assets/js/service.js"></script>
    <script src="./assets/js/index.js"></script>